src/cache/InvalidationPolicyCache.ts

```diff
@@ -380,6 +381,38 @@ export default class InvalidationPolicyCache extends InMemoryCache {
     return this._expire(true);
   }
 
+  readEntity(id: string) {
+    return this.entityTypeMap.readEntityById(id);
+  }
+
+  readEntityExpired(id: string) {
+    const entity = this._expire(true).find(
```

Comment:
To prevent us having to iterate through all entities to check expirations, which is somewhat more heavy, could we re-use the `readEntityExpirationTime` API and check against the current time to determine if it's expired here?

Reply:
Given that this is a boolean check I'd also recommend we go with a name like `isEntityExpired`